lib/tasks/pact.rake

```diff
@@ -1,5 +1,17 @@
 return if Rails.env.production?
 
 require "pact/tasks"
-require "pact_broker/client/tasks"
 require "pact/tasks/task_helper"
+
+desc "Verify that Collections honours all contracts in the pactfile created by a given branch of gds-api-adapters"
+task "pact:verify:branch", [:branch_name] => :environment do |t, args|
```

Comment:
I wonder if it's time to extract this code out into a common location, because I guess it should always be the same as: https://github.com/alphagov/publishing-api/blob/master/lib/tasks/pact.rake

Perhaps [govuk_test](https://github.com/alphagov/govuk_test) would be a good place for it.

Reply:
Yeah I think that's fair!